README.md

```diff
@@ -1,77 +1,19 @@
 # scikit-learn-knn-regression
 
-This package is in active development.
+> [!WARNING]  
+> This package is in active development!
 
-## Developer Guide
+`scikit-learn-knn-regression` (a.k.a. `sknnr`) is a package for running k-nearest neighbor imputation methods, including GNN (Ohmann & Gregory, 2002), MSN (Moeur & Stage 1995), and RFNN (Crookston & Finley, 2008), using estimators that are fully compatible with [`scikit-learn`](https://scikit-learn.org/stable/).
```

Comment:
I felt like we needed a list of 3 estimators, so I added RF-NN even though it's not ready not... Side-note, I think you usually refer to it as `RF-NN` instead of `RFNN`, but the [abbreviation system](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#abbreviations) doesn't work if there's a hyphen. Does this work as is? I suppose we could just make it two abbreviations of `RF` and `NN` instead.

Reply:
I think `RFNN` should be just fine.  And a good kick in the pants to get that one moving!